pages/login/login.css

```diff
@@ -0,0 +1,79 @@
+:root {
+  --login-form-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  --login-form-bg: white;
+  --login-form-btn-primary-bg: #007bff;
+  --login-form-btn-color: white;
+}
+
+/* login form container */
+
+.login-form-container {
+  margin-top: 7rem;
+  display: flex;
+  justify-content: center;
```

Comment:
For all the flex properties you can create a custom helper classes , just a suggestion